src/mobile/src/ui/views/wallet/History.js

```diff
@@ -320,64 +369,117 @@ class History extends Component {
             };
         });
 
-        return orderBy(formattedTransfers, 'time', ['desc']);
+        return { transactions: orderBy(formattedTransfers, 'time', ['desc']), totals };
     }
 
-    renderIOTATransactions() {
+    renderIOTATransactions(transactions) {
         const {
-            theme: { primary, body },
+            theme: { primary, body, input },
             t,
             isRefreshing,
         } = this.props;
-        const data = this.prepIOTATransactions();
-        const noTransactions = data.length === 0;
+        const { filter, search } = this.state;
+
+        const hasNoTransactions = search === '' && filter === 'All' && transactions.length === 0;
+        const hasNoFilteredTransactions = (filter !== 'All' || search !== '') && transactions.length === 0;
+
+        if (search === '!help') {
+            const searchOptions = {
+                XYZ: t('history:searchHelpText'),
+                '100': t('history:searchHelpValue'),
+                '100Mi': t('history:searchHelpUnits'),
+                '>100': t('history:searchHelpMore'),
+                '<100i': t('history:searchHelpLess'),
+            };
+            return (
+                <View style={[styles.searchHelpContainer, { backgroundColor: input.bg }]}>
+                    {map(keys(searchOptions), (item, index) => (
+                        <View keys={index} style={styles.searchHelpRow}>
```

Comment:
```suggestion
                        <View key={index} style={styles.searchHelpRow}>
```

The warning is still appearing. It looks like the property name should be `key` (see [React docs](https://reactjs.org/docs/lists-and-keys.html#keys))